collector/extension/solarwindsapmsettingsextension/extension.go

```diff
@@ -169,16 +170,24 @@ func refresh(extension *solarwindsapmSettingsExtension) {
 	}
 }
 
-func (extension *solarwindsapmSettingsExtension) Start(ctx context.Context, _ component.Host) error {
+func (extension *solarwindsapmSettingsExtension) Start(_ context.Context, _ component.Host) error {
 	extension.logger.Info("Starting up solarwinds apm settings extension")
-	ctx = context.Background()
+	ctx := context.Background()
 	ctx, extension.cancel = context.WithCancel(ctx)
-	var err error
-	extension.conn, err = grpc.Dial(extension.config.Endpoint, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})))
+	systemCertPool, err := x509.SystemCertPool()
```

Comment:
Use the certificates from system pool to make connection

Reply:
If i read the following code correctly the change is that you're explicitly setting the TLS trusted root certs to what was retrieved from system cert pool.  Is this solution something advised by AWS or found through trial and error?

Reply:
I see, so based on observation :)

Reply:
yes.